### DISCUSSION TITLE
Used datasets

### DISCUSSION THREAD

**Discussion by ophirazulai** [RapidAI/RapidTable#28] [Q&A]
Hi,

What is the model underneath RapidTable ? and on which datasets it was trained ?

Thanks
Ophir

**Comment by Joker1212** [comment]
it is from paddlex module, https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/module_usage/tutorials/ocr_modules/table_structure_recognition_en.md

**Comment by ophirazulai** [comment]
Thanks :-)
